environments/update_chs_configs.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+FILE="./env_template"
+CONFIG_DIR="../../chs-configs"
+APP_DIR="promise-to-file-web"
+
+update()
+{
+	if [ ${#environments[@]} -gt 0 ]; then 
+		for environment in "${environments[@]}"
+		do
+			path="$CONFIG_DIR/$environment/$APP_DIR/env"
+			if [ -f $path ]; then
+			    cat $FILE > $path
+			    echo "Replaced content of file $path with content";
+			else
+			    cat $FILE >> $path
+			    echo "Create new file $path";
+		    fi
+		done    
+	else 
+		echo "No config files updated or created please check content of $1"
+	fi
+}
+
+run()
+{
+	if [ "$1" = "dev" ]; then
+		 environments=()
+	     index=0
+	     while read line
+	     do
+	        environments[$index]=$line
+	        index=$((index+1))
+	     done < dev	
+	     update environments
+	elif [ "$1" = "live" ]; then
+		 environments=(environments)
+	     index=0
+		 while read line
```

Comment:
Indentation looks wrong on this line and the one in next block.

Reply:
Indentations fixed

Reply:
For some reason the indentation looks perculiar in the diffs here on git but in the script file they are now lined up.